### DISCUSSION TITLE
computeVariability in Signac

### DISCUSSION THREAD

**Discussion by Ryuichi30** [stuart-lab/signac#1473] [Q&A] [answered]
Hi,

In Signac, we can run chromVAR which returns motif activities (deviations) in a new assay.
In Signac, can we run "computeVariability", one of the chromVAR functions as well?

Thank you for your help!
Ryuichi

**Comment by timoast** [comment, marked as answer]
This is not supported in Signac, but you can always run ChromVAR yourself using functions in the [chromVAR package](https://bioconductor.org/packages/release/bioc/html/chromVAR.html)

**Comment by Ryuichi30** [comment]
Thanks. I will use ChromVAR.
